documentation/getstarted/advanced/migrationtesting.md

```diff
@@ -134,31 +143,42 @@ Lets create and set up our GitHub Actions workflow to test database migrations.
 
 This script accomplishes a few things. We are:
   * Installing Spawn to give us access to the `spawnctl` command from the pipeline agent
-  * Creating a [Spawn data container](https://www.spawn.cc/docs/concepts-data-container) from the data image we created locally named `Pagila:prod`
+  * Creating a [Spawn data container](https://www.spawn.cc/docs/concepts-data-container) from the data image we previously created named `Pagila:prod`
   * Using the official [Flyway docker image](https://hub.docker.com/r/flyway/flyway) to run `flyway migrate` on our data container, using migration scripts stored under the `sql` folder
   * Automating the cleanup of the database
 
-That's it - that is our migration test. We have quickly provisioned a database instance from our back up using Spawn, and set the Flyway connection details to point to that database and run the migration scripts in our repository. Any errors will be apparent here and show up on the developer's pull request that this ran against.
+That's it - that is our migration test. We have quickly provisioned a database instance from our back up using Spawn, and set the Flyway connection details to point to that database and run the migration scripts in our repository. Any errors will be apparent at this point before changes get merged into the master branch.
```

Comment:
Should probably reword `master branch` to say `main branch` or `default branch` (but go with `main` since that's what is used in the yaml 😛)

Reply:
thanks! 2d6b257